src/main/java/edu/greenblitz/tobyDetermined/subsystems/telescopicArm/Claw.java

```diff
@@ -30,15 +38,37 @@ public static void init(){
         instance = new Claw();
     }
 
-    public void grip() {
-        motor.set(MOTOR_POWER_GRIP);
+    public void cubeCatchMode (){
+        solenoid.set(DoubleSolenoid.Value.kReverse);
+        state = ClawState.CUBE_MODE;
+    }
+    public void coneCatchMode (){
+        solenoid.set(DoubleSolenoid.Value.kForward);
+        state = ClawState.CONE_MODE;
+    }
+
+    public void toggleSolenoid (){
+        if(solenoid.get() == DoubleSolenoid.Value.kReverse){
+            coneCatchMode();
+        }else{
+            cubeCatchMode();
+        }
     }
 
-    public void eject() {
-        motor.set(MOTOR_POWER_RELEASE);
+    public void motorGrip() {
+        motor.set(ControlMode.PercentOutput,RobotMap.telescopicArm.claw.MOTOR_POWER_GRIP);
+    }
+
+    public void motorEject() {
+        motor.set(ControlMode.PercentOutput,RobotMap.telescopicArm.claw.MOTOR_POWER_RELEASE);
     }
 
     public void stopMotor (){
-        motor.set(0);
+        motor.set(ControlMode.PercentOutput,0);
+    }
+
+    public enum ClawState{
```

Comment:
Something to describe if we have an item?

Reply:
we don't know if we will have a switch there

Reply:
I think, in the case, we don't - we should go the naive way and say - if we gripped already, we have an item - those states should be useful.

@yonnbro  what do you think?